src/openms/include/OpenMS/ANALYSIS/OPENSWATH/PeakIntegrator.h

```diff
@@ -517,6 +517,44 @@ namespace OpenMS
 
     void getDefaultParameters(Param& params);
 
+    /**
+      @brief Fit the given peak (a MSChromatogram) to the EMG peak model
+
+      It is desirable to use this method when dealing with saturated or cutoff peaks.
```

Comment:
Use “The method is able to recapitulate that actual peak area of saturated or cutoff peaks.  In additional, the method is able to fine tune the peak area of well acquired peaks.” Instead of “It is desirable to use this method when dealing with saturated or cutoff peaks.”